smpe/bld/psi/bld/prepare-PSI.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
```

Comment:
it will be easier to reuse existing smpe-*.sh code if the psi scripts are in the same dir as get-config.sh

Reply:
ok, I have moved psi script to the same directory